### ISSUE TITLE
Calibration upload 7a5773

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#1319]
Calibration images were uploaded to the directory that starts with “`7a5773_`”.

Please read the [workflow description](https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst) for further instructions about the calibration.

Additional comments by the uploader:

> Hi Torsten,
> 
> thanks for all your effort to this project, I really enjoy darktable (because of lensfun!) and I am glad to finally find a way to contribute also!
> 
> If my pictures aren't good please tell me, I'll go for another round.
> 
> Thanks and best regards aus Wien
> Bernhard
> 
> PS: This is fullframe with the second version of Canon's 24-105 L and I will upload pictures from an aps-c Camera as well.

**Comment by navpp** [comment]
Hi, 
I'm the uploader of the files. I already did some work on the calibration but I'm unsure if all I did was correct. I had troubles following the horizontal lines, since hugin draws its own white lines between the already selected points. This pretty much makes adding new points on the picture's horizontal line somewhat to a lottery for some pictures. At least it feels so while using the tool. Maybe the building in the pictures is not that suitable as I thought it would be.
Anyways, I'll use the result as it is for sure better than without any calibration. I attached the result files to this reply (It wasn't possible to attach tar.xz files generated by the script so manually ziped them)

I'm looking forward to someone checking on my results if they can be considered usable. 

Thanks and best regards
Bernhard

[lensfun_calibration.zip](https://github.com/lensfun/lensfun/files/5684995/lensfun_calibration.zip)
